### DISCUSSION TITLE
How to create dashboards from templates?

### DISCUSSION THREAD

**Discussion by HosAkh** [apache/superset#20112] [Q&A / Help]
Hi, 

I can't seem to locate this but is it possible to create templates for dashboards so if I want to start a new dashboard I can just select from one of the templates and point it toward the database or tables that I want it to use and it would just do the rest?

**Comment by kamalkeshavani-aiinside** [comment]
This is not available at the moment, you can request for it.

**Reply by faysal-ishtiaq** [reply]
Is there any update on this?
